### PR TITLE
Update bitfinex withdraw addr

### PR DESCRIPTION
**Pull request by speedmax**
Update supported withdraw types as per latest [bitfinex v1 API](https://docs.bitfinex.com/reference#rest-auth-withdrawal)

### CODE REVIEW

**Comment by kroitor** [comment]
Those currency codes should conform to other exchanges, there are several substitutions missed from here: could you rename the corresponding skipped currency codes as well (YYW, MNA and others) ? Thanks a lot!

File: js/bitfinex.js [hunk 1]
```diff
@@ -671,6 +671,29 @@ module.exports = class bitfinex extends Exchange {
             'AVT': 'aventus', // #1811
             'QTUM': 'qtum', // #1811
             'EDO': 'eidoo', // #1811
+
```
Comment (kroitor) [inline js/bitfinex.js]: This empty line should be deleted, as it breaks the transpiler rule from here:
https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#derived-exchange-classes

Apart from this empty line, the rest of the code is clean. Thx!

**Comment by kroitor** [comment]
Sorry, missed the link in the first comment: https://github.com/ccxt/ccxt/pull/2542/files#diff-a3a69516c164ff60bece197030b75589R219

**Comment by speedmax** [comment]
seems like your current `commonCurrencies` map has covered them all like SPANK coin

```
> b.client.currencies
{ AGI: { id: 'AGI', code: 'AGI', precision: 5 },
  AID: { id: 'AID', code: 'AID', precision: 5 },
  AIO: { id: 'AIO', code: 'AIO', precision: 5 },
  AVT: { id: 'AVT', code: 'AVT', precision: 5 },
  BAT: { id: 'BAT', code: 'BAT', precision: 5 },
  BCH: { id: 'BCH', code: 'BCH', precision: 5 },
  BFT: { id: 'BFT', code: 'BFT', precision: 5 },
  BTC: { id: 'BTC', code: 'BTC', precision: 5 },
  BTG: { id: 'BTG', code: 'BTG', precision: 5 },
  CFI: { id: 'CFI', code: 'CFI', precision: 5 },
  DAI: { id: 'DAI', code: 'DAI', precision: 5 },
  DASH: { id: 'DSH', code: 'DASH', precision: 5 },
  DATA: { id: 'DAT', code: 'DATA', precision: 5 },
  EDO: { id: 'EDO', code: 'EDO', precision: 5 },
  ELF: { id: 'ELF', code: 'ELF', precision: 5 },
  EOS: { id: 'EOS', code: 'EOS', precision: 5 },
  ETC: { id: 'ETC', code: 'ETC', precision: 5 },
  ETH: { id: 'ETH', code: 'ETH', precision: 5 },
  ETP: { id: 'ETP', code: 'ETP', precision: 5 },
  EUR: { id: 'EUR', code: 'EUR', precision: 5 },
  FUN: { id: 'FUN', code: 'FUN', precision: 5 },
  GBP: { id: 'GBP', code: 'GBP', precision: 5 },
  GNT: { id: 'GNT', code: 'GNT', precision: 5 },
  IOS: { id: 'IOS', code: 'IOS', precision: 5 },
  IOTA: { id: 'IOT', code: 'IOTA', precision: 5 },
  JPY: { id: 'JPY', code: 'JPY', precision: 5 },
  LRC: { id: 'LRC', code: 'LRC', precision: 5 },
  LTC: { id: 'LTC', code: 'LTC', precision: 5 },
  MANA: { id: 'MNA', code: 'MANA', precision: 5 },
  MTN: { id: 'MTN', code: 'MTN', precision: 5 },
  NEO: { id: 'NEO', code: 'NEO', precision: 5 },
  ODE: { id: 'ODE', code: 'ODE', precision: 5 },
  OMG: { id: 'OMG', code: 'OMG', precision: 5 },
  QASH: { id: 'QSH', code: 'QASH', precision: 5 },
  QTUM: { id: 'QTM', code: 'QTUM', precision: 5 },
  RCN: { id: 'RCN', code: 'RCN', precision: 5 },
  RDN: { id: 'RDN', code: 'RDN', precision: 5 },
  REP: { id: 'REP', code: 'REP', precision: 5 },
  REQ: { id: 'REQ', code: 'REQ', precision: 5 },
  RLC: { id: 'RLC', code: 'RLC', precision: 5 },
  RRT: { id: 'RRT', code: 'RRT', precision: 5 },
  SAN: { id: 'SAN', code: 'SAN', precision: 5 },
  SNGLS: { id: 'SNG', code: 'SNGLS', precision: 5 },
  SNT: { id: 'SNT', code: 'SNT', precision: 5 },
  SPANK: { id: 'SPK', code: 'SPANK', precision: 5 },
  TNB: { id: 'TNB', code: 'TNB', precision: 5 },
  TRX: { id: 'TRX', code: 'TRX', precision: 5 },
  USD: { id: 'USD', code: 'USD', precision: 5 },
  WAX: { id: 'WAX', code: 'WAX', precision: 5 },
  XMR: { id: 'XMR', code: 'XMR', precision: 5 },
  XRP: { id: 'XRP', code: 'XRP', precision: 5 },
  YOYOW: { id: 'YYW', code: 'YOYOW', precision: 5 },
  ZEC: { id: 'ZEC', code: 'ZEC', precision: 5 },
  ZRX: { id: 'ZRX', code: 'ZRX', precision: 5 } }
> 

```

**Comment by kroitor** [comment]
> seems like your current commonCurrencies map has covered them all like SPANK coin

Yes, but in code you submitted, you used YYW instead of YOYOW, and MNA instead of MANA... I mean, could you sync your withdrawal currency codes with commonCurrencies ? Thank you!

**Pull request merged**